### PR TITLE
Add header constants

### LINKED ISSUE #5: Add constants to maeparser headers
When integrating maeparser with RDKit, @mbeachy raised the point that we should probably move string constants like 'f_m_ct' into headers in maeparser, where we can name them in obvious ways that make them look less-magicky in lilbrary-user code.  The obvious things we could constant:

- f_m_ct
- s_m_title
- m_atom
- r_m_*_coord
- m_bond
- i_m_from / i_m_to / i_m_order
- i_m_formal_charge (maybe some other common properties?)

### PR DESCRIPTION
**Pull request by lorton**
fixes #5

### CODE REVIEW

**Comment by lorton** [comment]
@ricrogz - it looks like our window compiler setting here doesn't support constexpr...which is very weird since that's supposed to be C++11.  Any idea?

**Comment by ricrogz** [comment]
Yes: constexpr is not available for VS2013:

https://msdn.microsoft.com/en-us/library/hh567368.aspx

File: MaeConstants.hpp [hunk 1]
```diff
@@ -0,0 +1,25 @@
+#pragma once
+#include <string>
+
+namespace schrodinger
+{
+namespace mae
+{
+
+constexpr const char* MAE_FORMAT_VERSION = "s_m_m2io_version";
```
Comment (ricrogz) [inline MaeConstants.hpp]: Maybe `const char* const` ?

File: README.md [hunk 2]
```diff
@@ -6,7 +6,8 @@ maeparser is a parser for [Schrodinger](https://www.schrodinger.com/) Maestro
 files.
 
 Structure files (.mae,.maegz,.mae.gz) can contain multiple structures
-deliminated by "f_m_ct".
+deliminated by "f_m_ct".  See [MaeConstants.hpp](MaeConstants.hpp)
```
Comment (ricrogz) [inline README.md]: delimited

**Pull request merged**